expense_app/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from django.utils import timezone
+# Create your models here.
+
+
+class Expense(models.Model):
+    title = models.CharField(max_length=50)
+    budget = models.IntegerField()
+    date = models.DateTimeField(default=timezone.now)
+    # assign = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Why is this commented out?

Reply:
the User model is not merged yet , how do i remove the comment and continue to work on my model without waiting for the User model to be merged?